mono_repo/lib/src/root_config.dart

```diff
@@ -81,7 +81,7 @@ class RootConfig extends ListBase<PackageConfig> {
 
   RootConfig._(this.rootDirectory, this.monoConfig, this._configs);
 
-  factory RootConfig({String rootDirectory, bool recursive = false}) {
+  factory RootConfig({String rootDirectory, bool recursive = true}) {
     rootDirectory ??= p.current;
 
```

Comment:
It's preferable to do `recursive ??= true`

Reply:
yup!